### PR TITLE
Arodriguez/feature 153 report hazards display the hazard title depending on the category selected

### PR DESCRIPTION
**Pull request by ALFAROSO**
Display the hazard title or question dynamically just 5 new lines also the Back button

### CODE REVIEW

**Review by ReinhardtBotha — CHANGES_REQUESTED** [review]
Nice Alberth, just some things I noticed: 

1. When I select Wildfire, the back button doesn't seem to work.
2. I'm able to click continue even when no hazard category or option is selected, but they should be mandatory.

File: pages/report-hazard/index.html [hunk 1]
```diff
@@ -116,7 +116,7 @@ <h1>
 						<button id="starCameraBtn" type="button" style="display:none"></button>
 						<!-- TODO// Add icon here -->
 						<span>Take a picture</span>
-						<span>at the hazard site? Click a picture to add in the report.</span>
+						<span>At the hazard site? Take a picture to add in the report.</span>
```
Comment (whamester) [inline pages/report-hazard/index.html:119]: this At should be at

File: pages/report-hazard/index.html [hunk 2]
```diff
@@ -147,7 +147,7 @@ <h1>
 							style="display:none;" />
 						<!-- TODO// Add icon here -->
 						Take a picture<br>
-						<span>At the hazard site? Click a picture to add in the report.</span>
+						<span>At the hazard site? Take a picture to add in the report.</span>
```
Comment (whamester) [inline pages/report-hazard/index.html:150]: same: At -> at

File: pages/report-hazard/script.js [hunk 3]
```diff
@@ -849,3 +854,31 @@ delete3Element.addEventListener('click', () => {
...
+  const currentIndex = array.indexOf(currentHash);
+
+  if (currentIndex > 0) {
+    const previousIndex = currentIndex - 1;
+    const previousHash = array[previousIndex];
+
+    url.hash = previousHash;
+    window.location.href = url.href;
+  } else {
+    window.location.href = 'http://127.0.0.1:5500/pages/home/';
```
[the remark this answers is not in the document]
Comment (whamester) [reply pages/report-hazard/script.js:882]: Good catch!
also, if you could redirect to /pages/home/index.html, so the transition between the pages looks smoother

File: pages/report-hazard/style.css [hunk 4]
```diff
@@ -23,3 +23,16 @@ img {
 	width: 10%;
 	height: auto;
 }
+
+.desktop{
+	display: none;
+}
+
+@media screen and (min-width: 800px){
```
Comment (whamester) [inline pages/report-hazard/style.css:31]: Nice work having the media queries take care of the images section, did you remove the prev section of the code? just double-check so we don't have code in two places doing the same

File: pages/report-hazard/script.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ALFAROSO) [reply pages/report-hazard/script.js:882]: done

File: pages/report-hazard/index.html (hunk 1 above)
Comment (ALFAROSO) [reply pages/report-hazard/index.html:119]: done

File: pages/report-hazard/index.html (hunk 2 above)
Comment (ALFAROSO) [reply pages/report-hazard/index.html:150]: done

File: pages/report-hazard/script.js [hunk 5]
```diff
@@ -261,8 +261,10 @@ const getCategories = async () => {
         currentReport.category.name = category.name;
 
         const options = selectedCategory.options ?? [];
+        const electedOptionQuestion =
+          selectedCategory.ui_settings.report_hazard_question ?? [];
 
-        populateHazardOptions(options);
+        populateHazardOptions(options, electedOptionQuestion);
```
[the remark this answers is not in the document]
Comment (ALFAROSO) [reply pages/report-hazard/script.js:267]: done

File: pages/report-hazard/style.css (hunk 4 above)
Comment (ALFAROSO) [reply pages/report-hazard/style.css:31]: done

**Review by whamester — APPROVED** [review]

**Review by ReinhardtBotha — APPROVED** [review]
good job!

**Pull request merged**